src/main/scala/com/rbmhtechnology/calliope/serializer/PayloadSerializer.scala

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2015 - 2017 Red Bull Media House GmbH <http://www.redbullmediahouse.com> - all rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.rbmhtechnology.calliope.serializer
+
+import akka.actor.ExtendedActorSystem
+import akka.serialization.{NullSerializer, SerializationExtension, SerializerWithStringManifest}
+import com.google.protobuf.ByteString
+import com.rbmhtechnology.calliope.serializer.CommonFormats.PayloadFormat
+
+import scala.util.{Failure, Success, Try}
+
+trait PayloadSerializer {
+  def payloadFormatBuilder(payload: AnyRef): Try[PayloadFormat.Builder]
+  def payload(payloadFormat: PayloadFormat): Try[AnyRef]
```

Comment:
Why wrapping into `Try` if you anyway force a `.get` in `fromBinary` and `toBinary` in serializers? These methods could throw exceptions directly so that you save the wrapper overhead (this might be relevant as de/serializations are frequent operations).